JenkinsfileRT

```diff
@@ -25,56 +25,50 @@ data_config.server_id = 'bytesalad'
 data_config.root = 'clone/tests_output'
 data_config.match_prefix = '(.*)_result' // .json is appended automatically
 
+// Test 1 ///////////////////////////
 bc1 = new BuildConfig()
 bc1.nodetype = 'linux'
 bc1.env_vars = ['TEST_BIGDATA=https://bytesalad.stsci.edu/artifactory']
 bc1.name = '3.10'
 bc1.conda_packages = ['python=3.10']
 bc1.build_cmds = ["pip install numpy astropy pytest-cov ci-watson",
-		 "pip install --upgrade -e '.[test]'",
-                 "pip freeze"]
+                "pip install --upgrade -e '.[test]'",
```

Comment:
Can we excise this `\t` character? ... Oh the PR merged while I was writing this.